### PR TITLE
feat: add support for en and fr locale

### LINKED ISSUE #13: Supported languages should be French and English
This should be reflected at:

- Content structure level
- Navigation level
- Language switch level

### PR DESCRIPTION
**Pull request by ramboz**
Adding support for multiple languages on the site (fr + en), including:
- localized navigation
- localized pages in different folders
- proper switching to a matching page in the other language
- logo title in the header

Screenshot 2022-11-23 at 12 18 30 PM Screenshot 2022-11-23 at 12 18 36 PM
Screenshot 2022-11-23 at 12 18 52 PM

Fix #13

Test URLs:
- Before: https://main--rolex-event--hlxsites.hlx.page/
- After: https://issue-13--rolex-event--hlxsites.hlx.page/en/

### CODE REVIEW

**Comment by meryllblanchet** [comment]
Reminder: the logo needs to be updated for 2023.

**Comment by ramboz** [comment]
@meryllblanchet As far as I understand, we don't have it yet?

File: scripts/scripts.js [hunk 1]
```diff
@@ -75,7 +75,8 @@ export function decorateMain(main) {
  * loads everything needed to get to LCP.
  */
 async function loadEager(doc) {
-  document.documentElement.lang = 'en';
+  const [, lang] = window.location.pathname.split('/');
+  document.documentElement.lang = lang && lang.length === 2 ? lang : 'en';
```
Comment (meryllblanchet) [inline scripts/scripts.js:79]: Let's confirm that the default is english (and not french)

Comment (ramboz) [reply scripts/scripts.js:79]: Let's track this in #16

**Pull request merged**